kuwala/core/backend/app/models/demo/README.md

```diff
@@ -0,0 +1,112 @@
+
+
+## Prerequisites
+1) Install [R](https://www.r-project.org/) (tested on version R 4.1.3).
+
+2) Once R is installed, install `remote` package library to R so we can install Robyn via remote github repository. Do this by going to your terminal, call `R`, then `install.packages('remote')`.
+
+3) Still in the R console, install `reticulate` package by inserting `install.packages("reticulate")` so later Robyn can use Python functions in it.
+
+4) Install `nevergrad` in `reticulate`. In R console, insert: 
+```R
+virtualenv_create("r-reticulate")
+use_virtualenv("r-reticulate", required = TRUE)
+py_install("nevergrad", pip = TRUE)
+```
+Then check Python in R configuration:
+```R
+py_config() 
+
+## In case nevergrad still can't be installed,
+Sys.setenv(RETICULATE_PYTHON = "~/.virtualenvs/r-reticulate/bin/python")
+# Reset your R session and re-install Nevergrad
+
+```
+**or alternatively**, create virtual environment using `conda`:
+```R
+conda_create("r-reticulate", "Python 3.9") # Only works with <= Python 3.9 sofar
+use_condaenv("r-reticulate")
+conda_install("r-reticulate", "nevergrad", pip=TRUE)
+# In case nevergrad still can't be installed,
+# please locate your python file and run this line with your path:
+use_python("~/Library/r-miniconda/envs/r-reticulate/bin/python3.9")
+# Alternatively, force Python path for reticulate with this:
+Sys.setenv(RETICULATE_PYTHON = "~/Library/r-miniconda/envs/r-reticulate/bin/python3.9")
+# Finally, reset your R session and re-install Nevergrad with option 2
+
+```
+5) Install Robyn, still in the R console, insert: `remotes::install_github("facebookexperimental/Robyn/R")`, it will install dependencies as well. 
+
+6) Install required libraries in `kuwala/core/backend/requirements.txt`. You can install from this directory using:
+```
+pip install -r ../../../requrements.txt
+``` 
+
+## Database
+This automation tested by importing Robyn's sample data from database. The sample data exported to separate tables containing each marketing channels. In order to run properly, these tables needed to be exist in the database. Those tables and row names are:
```

Comment:
Should be "This automation has been tested by […] The sample data was exported […], these tables need to exist in the database. […].